### PR TITLE
MINOR: log when controller begins processing logdir failure event

### PR DESCRIPTION
**Pull request by dhruvilshah3**

### CODE REVIEW

**Comment by dhruvilshah3** [comment]
@junrao do you think it makes sense to add this logging when the controller starts processing the log dir failure event?

File: core/src/main/scala/kafka/controller/KafkaController.scala [hunk 1]
```diff
@@ -337,9 +337,12 @@ class KafkaController(val config: KafkaConfig, zkClient: KafkaZkClient, time: Ti
    * This callback is invoked by the controller's LogDirEventNotificationListener with the list of broker ids who
    * have experienced new log directory failures. In response the controller should send LeaderAndIsrRequest
    * to all these brokers to query the state of their replicas
+   * to all these brokers to query the state of their replicas. Replicas with an offline log directory respond with
+   * KAFKA_STORAGE_ERROR.
```
Comment (junrao) [inline core/src/main/scala/kafka/controller/KafkaController.scala]: Perhaps we can change it slightly to "Replicas with an offline log directory respond with KAFKA_STORAGE_ERROR, which will be handled by the LeaderAndIsrResponseReceived event"?

File: core/src/main/scala/kafka/controller/KafkaController.scala [hunk 2]
```diff
@@ -337,9 +337,12 @@ class KafkaController(val config: KafkaConfig, zkClient: KafkaZkClient, time: Ti
...
    */
   private def onBrokerLogDirFailure(brokerIds: Seq[Int]) {
     // send LeaderAndIsrRequest for all replicas on those brokers to see if they are still online.
+    info(s"Handling log directory failure for ${brokerIds.mkString(",")}")
```
Comment (junrao) [inline core/src/main/scala/kafka/controller/KafkaController.scala]: failure for  => failure for brokers ?

**Review by junrao — COMMENTED** [review]
@dhruvilshah3 Thanks for the patch. Makes sense. Just a couple of minor comments.

**Comment by dhruvilshah3** [comment]
@junrao thanks for the quick review. I pushed a commit to address your comments.

**Review by junrao — APPROVED** [review]
@dhruvilshah3 : Thanks for the updated patch. LGTM

**Pull request merged**